pkg/apis/org/v1/checlusterrestore_types.go

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+// Important: when any changes are made, CRD must be regenerated.
+// Please use olm/update-resources.sh script for that.
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CheClusterRestoreSpec defines the desired state of CheClusterRestore
+type CheClusterRestoreSpec struct {
+	// If true, deletes the CR after successful restore.
+	DeleteConfigurationAfterRestore bool `json:"deleteConfigurationAfterRestore,omitempty"`
```

Comment:
```suggestion
```

Reply:
I would keep it, but since you insist, I'll remove this field.